magentov2/plugin/Indodana/PayLater/Gateway/Http/Client/ClientMock.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Copyright © 2016 Magento. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Indodana\Paylater\Gateway\Http\Client;
+
+use Magento\Payment\Gateway\Http\ClientInterface;
+use Magento\Payment\Gateway\Http\TransferInterface;
+use Magento\Payment\Model\Method\Logger;
+
+class ClientMock implements ClientInterface
```

Comment:
Can you explain in general why you need this class?

Please add it on code comment

Reply:
it's part of component of intaraction in payment gateway process

![image](https://user-images.githubusercontent.com/32725392/93152880-10288700-f72a-11ea-8270-832f41301ead.png)
`

https://devdocs.magento.com/guides/v2.4/payments-integrations/payment-gateway/payment-gateway-structure.html`